File: Pension-Management-Backend/Pensioner-Detail-Microservice/src/main/java/com/cognizant/pensioner/detail/model/PensionerDetail.java
package com.cognizant.pensioner.detail.model;

import java.sql.Date;

import javax.persistence.Embedded;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class PensionerDetail {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	private String aadhaarNumber;
	private String name;
	private Date dateOfBirth;
	private String panNumber;
	private Double salary;
	private Double allowance;
	private String pensionType;
	@Embedded
	private BankDetail bankDetail;
	
}

File: Pension-Management-Frontend/Pension-Management-Portal/src/app/model/pensioner-detail.ts
export class PensionerDetail {
    public id: number;
	public name: string;
	public dateOfBirth: Date;
	public panNumber: string;
    public aadhaarNumber: string;
	public salary: number;
	public allowance: number;
	public pensionType: string;

    constructor() {
        this.id = 0;
        this.name = '';
        this.dateOfBirth = new Date();
        this.panNumber = '';
        this.aadhaarNumber = '';
        this.salary = 0;
        this.allowance = 0;
        this.pensionType = '';
    }
}